tests/assemblers/test_xgboost.py

```diff
@@ -10,7 +10,7 @@ def test_binary_classification():
                                       max_depth=1)
     utils.train_model_classification_binary(estimator)
 
-    assembler = assemblers.XGBoostModelAssembler(estimator)
+    assembler = assemblers.XGBoostTreeModelAssembler(estimator)
```

Comment:
I guess it should be `XGBoostModelAssemblerSelector` (here and all other occurrences of assemblers in this file) to increase the code coverage.

Reply:
Makes sense. I can do that 👍